.lintstagedrc

```diff
@@ -4,5 +4,5 @@
     "prettier --write",
     "jest --findRelatedTests"
   ],
-  "*.md": ["prettier --write"]
+  "*.{md,yml}": ["prettier --write"]
```

Comment:
I think we need to include `json` here right? It's the only file type missing from our scripts.